src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -1311,13 +1311,42 @@ private static object FoldDecimalBinaryOperators(BinaryOperatorKind kind, Consta
             return null;
         }
 
+        private static object FoldNativeIntegerBinaryOperator(BinaryOperatorKind kind, ConstantValue valueLeft, ConstantValue valueRight)
```

Comment:
>FoldNativeIntegerBinaryOperator [](start = 30, length = 31)

Given the name, one might assume this method handles all operators, but this is not the case. Consider reflecting this fact in the name. FoldNativeIntegerOverflowingBinaryOperator or something similar.